src/components/create-account/WeeklyInfo/WeeklyInfo.jsx

```diff
@@ -218,10 +220,12 @@ const WeeklyInfo = ({ availability, setAvailability }) => {
 
   return (
     <div className="app">
-      <h1>Weekly Availability</h1>
-      <div className="row">
-        <div className="mixed-chart"> {renderChart()} </div>
-      </div>
+      <center>
+        <h1 className={styles.heading}>Weekly Availability</h1>
+        <div className="row">
+          <div className="mixed-chart"> {renderChart()} </div>
```

Comment:
Any reason why these class names are quoted instead of from module.css? Are these classes from antdesign or somewhere else 🤔

Reply:
It seems like these class names aren't used anywhere and are just for easier readability/labeling. I'll just remove them.